exchange/exchange.go

```diff
@@ -303,6 +303,7 @@ func validateAndNormalizeDealTier(impDeal *DealTier) bool {
 func updateHbPbCatDur(bid *pbsOrtbBid, dealTierInfo *DealTierInfo, bidCategory map[string]string) {
 	if bid.dealPriority >= dealTierInfo.MinDealTier {
```

Comment:
Looking  function name - `updateHbPbCatDur`, Can we move the following condition outside of this function
`if bid.dealPriority >= dealTierInfo.MinDealTier {`

after change, It will look like 
<img width="1049" alt="image" src="https://user-images.githubusercontent.com/9314645/97535546-cea22180-19e1-11eb-8319-4c2c88ac99d8.png">